saiplayer/Makefile.am

```diff
@@ -8,6 +8,15 @@ else
 DBGFLAGS = -g
 endif
 
+noinst_LIBRARIES = libSaiPlayer.a
+libSaiPlayer_a_SOURCES = \
+						 CommandLineOptions.cpp \
+						 CommandLineOptionsParser.cpp \
+						 SaiPlayer.cpp
+
+
+libSaiPlayer_a_CPPFLAGS = $(DBGFLAGS) $(AM_CPPFLAGS) $(CFLAGS_COMMON) -std=c++14
+
 saiplayer_SOURCES = saiplayer.cpp
-saiplayer_CPPFLAGS = $(DBGFLAGS) $(AM_CPPFLAGS) $(CFLAGS_COMMON)
-saiplayer_LDADD = -lhiredis -lswsscommon -lpthread -L$(top_srcdir)/meta/.libs -lsaimetadata -lsaimeta -L$(top_srcdir)/lib/src/.libs -lsairedis
+saiplayer_CPPFLAGS = $(DBGFLAGS) $(AM_CPPFLAGS) $(CFLAGS_COMMON) -std=c++14
+saiplayer_LDADD = libSaiPlayer.a ../syncd/libSyncd.a ../lib/src/libSaiRedis.a -lhiredis -lswsscommon -lpthread -L$(top_srcdir)/meta/.libs -lsaimetadata -lsaimeta 
```

Comment:
../syncd/libSyncd.a looks like this breaks the lgtm build process, @kcudnik, can you check?

Reply:
it needs #599 which is already merged, how to restart lgtm to check again ?

Reply:
can you do a rebase and push?

Reply:
![image](https://user-images.githubusercontent.com/2293967/79005492-d1076180-7b0b-11ea-868d-8009481fb13e.png)

Reply:
re run somehow don;t work when i click it